files/build_templates/init_cfg.json.j2

```diff
@@ -133,5 +133,16 @@
             "digits_class": "true",
             "special_class": "true"
         }
+    },
+    "SYSTEM_DEFAULTS" : {
+{%- if include_mux == "y" %}
```

Comment:
Is it better to generate the `SYSTEM_DEFAULTS` table only when `sonic_asic_platform == "mellanox"`, and `include_mux == "y"`? In this way, there will be no change for other platforms.

Reply:
Thank you for your remark.
I thought about it previously. But I decided that it may look confusing that **SYSTEM_DEFAULT** table (that is generic table) is under "mellanox" and "mux" condition.
What do you think about it?

Reply:
Make sense. Please also consider when we are upgrading from previous version, the `mux_tunnel_egress_acl` is absent. Our code should be handle that exception.